app/uk/gov/hmrc/homeofficesettledstatusstubs/models/StatusCheckResult.scala

```diff
@@ -19,6 +19,7 @@ package uk.gov.hmrc.homeofficesettledstatusstubs.models
 import java.time.LocalDate
 import play.api.libs.json.{Format, Json}
 
+//todo immigrationCheckResult
```

Comment:
I don't understand this todo

Reply:
I think i wanted to rename it, tbh im not sure anymore 😂  will remove

Reply:
:+1: